src/docfx/build/metadata/ContributionProvider.cs

```diff
@@ -108,6 +108,14 @@ internal class ContributionProvider
             return contributionInfo;
         }
 
+        public string?[]? GitContributorInformation(ErrorBuilder errors, FilePath file)
+        {
+            return GetContributionInfo(errors, file, new SourceInfo<string>(""), true)?.Contributors?.
```

Comment:
can cache the `contributors` in line#75.
don't have to run the whole process twice. Or you can have the `GetContributionInfo` return the two entities.